benchmarks/build.sbt

```diff
@@ -20,7 +20,7 @@ scalaVersion := "2.12.14"
 lazy val root = (project in file("."))
   .settings(
     name := "benchmarks",
-    libraryDependencies += "org.apache.spark" %% "spark-sql" % "3.2.0" % "provided",
```

Comment:
QQ - why switch to Spark 3.1.2 vs. Spark 3.2.0?   Was this an issue related to Dataproc?

Reply:
Got it - makes sense - thanks!